examples/borer-circe-migration/src/main/scala/btest/Test1.scala

```diff
@@ -0,0 +1,25 @@
+package pl.marconzet
+package btest
+
+import io.bullet.borer.{Codec, Json}
+import io.bullet.borer.derivation.ArrayBasedCodecs.{deriveAllCodecs, deriveCodec}
+
+object Test1 extends App {
```

Comment:
Hmm instead of `Test1`... maybe `BasicCodecs`? or anyway, sth that explain the purpose more clearly

Reply:
Also, do we even need this example? since no migration is involved here, it's a Borer-only piece